src/OpenTelemetry.Exporter.OpenTelemetryProtocol/Implementation/SpanDataExtensions.cs

```diff
@@ -0,0 +1,264 @@
+﻿// <copyright file="SpanDataExtensions.cs" company="OpenTelemetry Authors">
+// Copyright 2018, OpenTelemetry Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+// </copyright>
+
+using System;
+using System.Collections.Generic;
+using System.Linq;
+
+using Google.Protobuf;
```

Comment:
I'm not sure about the preferred pattern regarding the organization of the using directives. Let me know the preferred style.

Reply:
as long as Stylecop is happy =)